source/_integrations/shelly.markdown

```diff
@@ -38,9 +44,19 @@ We recommend using `unicast` for communication. To enable this, enter the local
 Integration is communicating directly with the device; cloud connection is not needed.
 </div>
 
-{% include integrations/config_flow.md %}
+## CoAP port (generation 1)
```

Comment:
This is a very rare case, let's keep it at at the bottom of the docs

Reply:
Done